grammar_test/semicolon_insertion.txt

```diff
@@ -61,8 +61,7 @@ function c() {return d}
 (program
   (expression_statement
     (function (identifier) (formal_parameters) (statement_block
-      (trailing_expression_statement (identifier)))))
-  (expression_statement
```

Comment:
Fixture:

```javascript
function a() {b}
function c() {return d}
```

This patch means these functions are treated in sequence, and are regarded as a single `expression_statement`. Curious what @maxbrunsfeld thinks about this, and if this is an acceptable tradeoff for the relative small impact this change represents to the parser?